### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by runocan**

### CODE REVIEW

File: lib/login-buttons.dart [hunk 1]
```diff
@@ -7,6 +8,10 @@ class LoginButton extends StatelessWidget {
   final String text;
   final double padding;
 
+  Future navigateToSignupPage(context) async {
+    Navigator.push(context, MaterialPageRoute(builder: (context) => SignupPage()));
+  }
+
```
Comment (coolfortune) [inline lib/login-buttons.dart]: Any reason this function is defined inside login-register.dart? I feel like this should be defined in main.dart instead, then passed into a LoginButton widget

**Review by coolfortune — CHANGES_REQUESTED** [review]
Any reason this function is defined inside login-register.dart? I feel like this should be defined in main.dart instead, then passed into a LoginButton widget

File: lib/login-buttons.dart (hunk 1 above)
Comment (runocan) [reply lib/login-buttons.dart]: Was just testing out the function. Will move into main.dart!

Comment (coolfortune) [reply lib/login-buttons.dart]: Cool, dont forget to push your changes

**Pull request merged**